windows/wintools_uninstall_verify/prepare_pvscsi_vmxnet3_device.yml

```diff
@@ -10,44 +10,49 @@
     vm_has_pvscsi_ctl: false
     vm_has_vmxnet3_net: false
 
-# Check if boot disk controller is PVSCSI
-- name: "Set fact of VM with PVSCSI boot disk controller"
+# PVSCSI boot disk controller
+- name: "Set boot disk controller type to PVSCSI for new VM"
   ansible.builtin.set_fact:
     vm_has_pvscsi_boot_disk: true
   when:
     - new_vm is defined and new_vm | bool
     - boot_disk_controller is defined and boot_disk_controller == 'paravirtual'
-- name: "Get boot disk controller type for existing VM"
+
+- name: "Set boot disk controller type for existing VM"
   block:
-    - include_tasks: ../utils/win_get_boot_disk_ctl_type.yml
-    - name: "Set fact of VM with PVSCSI boot disk controller"
+    - name: "Get boot disk controller type in guest OS"
+      include_tasks: ../utils/win_get_boot_disk_ctl_type.yml
+    - name: "Set boot disk controller type to PVSCSI"
       ansible.builtin.set_fact:
         vm_has_pvscsi_boot_disk: true
       when:
         - win_boot_disk_ctl_type == 'paravirtual'
   when: new_vm is undefined or not new_vm | bool
```

Comment:
`when` can be moved before `block`, and same to below `block` tasks

Reply:
Changed, thanks.